applications/nrf5340_audio/unicast_client/main.c

```diff
@@ -326,7 +326,7 @@ static void bt_mgmt_evt_handler(const struct zbus_channel *chan)
 
 		ret = bt_r_and_c_discover(msg->conn);
 		if (ret) {
-			LOG_WRN("Failed to discover rendering services");
+			LOG_WRN("Failed to discover rendering and capture services");
```

Comment:
Unrelated to the changes in this PR, but consider moving the capture and rendering control out of the module called `unicast_client`. In CAP the unicast client is part of the CAP initiator and does not handle volume control (or microphone control), but that's the behavior of the CAP commander, which does not do stream control (and thus not a unicast client).